app/views/families/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<section class="section facilities__index">
+  <div class="container facilities__index__container">
+    <p id="notice"><%= notice %></p>
+
+    <div class="facilities__index__container__header">
+      <h1 class="title">Families</h1>
+      <%= link_to new_family_path, class: "button-primary" do %>
+        New Family <%= image_tag  "icons/plus.png", size: 17 %>
+      <% end %>
+    </div>
+
+    <table class="table  table-hover table-responsive-sm">
+      <thead>
+      <tr class="bg-dark-light">
+        <th>Name</th>
+        <th>Female</th>
+        <th>Male</th>
+        <th>Tank</th>
+        <th colspan="3"></th>
+      </tr>
+      </thead>
+
+      <tbody>
+      <% @families.each do |family| %>
+        <tr>
+          <td><%= family.name %></td>
+          <td><%= link_to family.female.pii_tag, animal_path(family.female) %></td>
+          <td><%= link_to family.male.pii_tag, animal_path(family.male) %></td>
```

Comment:
Thoughts on adding delagation to `Family` that will give us `male_pii_tag` and `female_pii_tag` methods on family?

Reply:
Sounds good @benreyn. This is going to be the primary identifier for end user use, right? What would the `tag_id` be?

Reply:
Im working on teasing that out from the data we have from our stakeholders, and digging through legacy code from when I joined the project. Ill follow up on this ASAP.